### PR TITLE
feat: added thanks modal to confirm page

### PR DESCRIPTION
**Pull request by elpollo12**
### Description

Add modal to confirm page. Currently appears when entering the page

fixes # (issue)

#46 

#### Screenshots

Captura de pantalla de 2021-10-15 09-37-07

### CODE REVIEW

**Comment by Israel-Laguan** [comment]
Please rebase

**Comment by Israel-Laguan** [comment]
Can you center the content in the modal, please?
image

File: src/components/Text.jsx [hunk 1]
```diff
@@ -24,35 +24,42 @@ const Text = styled.p`
...
         color: #00000029;
       }
     `}
+
+    /* Align */
+  ${props =>
+    props.center &&
+    css`
+      ${text.textCenter};
+    `}
```
Comment (Israel-Laguan) [inline src/components/Text.jsx]: This was already implemented in Ginna's PR, please rebase

File: src/pages/Confirm.jsx [hunk 2]
```diff
@@ -49,16 +51,14 @@ const Confirm = () => {
...
-      <div className="thanks-modal">
-        <img src={close} alt="Close icon" />
-        <img src={handshake} alt="Handshake" />
-        <Text as="h1" color="primary" small>
-          ¡Gracias por confiar en nosotros!
-        </Text>
-        <Text as="button" bold uppercase>
-          Aceptar
+      <Modal isOpen={isModalOpen} handleClick={() => changeIsModalOpen(false)} actionText="Aceptar">
+        <div style={{margin:'auto 0 auto 0'}}>
```
Comment (Israel-Laguan) [inline src/pages/Confirm.jsx]: Please remove this inline style and transform to styled component. An alternative is to create a global class

**Comment by elpollo12** [comment]
Rebase and Fix Modal Content Ready

File: src/pages/Confirm.jsx (hunk 2 above)
Comment (elpollo12) [reply src/pages/Confirm.jsx]: Inline style was removed

**Comment by Israel-Laguan** [comment]
I made a little change in style, please check

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**